model/event/DeliveryCreatedEvent.php

```diff
@@ -17,18 +17,31 @@
  * Copyright (c) 2016 (original work) Open Assessment Technologies SA
  *
  */
+
 namespace oat\taoDeliveryRdf\model\event;
 
+use core_kernel_classes_Resource;
 use oat\tao\model\webhooks\WebhookSerializableEventInterface;
+use \core_kernel_persistence_Exception;
 
 class DeliveryCreatedEvent extends AbstractDeliveryEvent implements WebhookSerializableEventInterface
 {
+    const ASSEMBLED_DELIVERY_ORIGIN_URI = 'http://www.tao.lu/Ontologies/TAODelivery.rdf#AssembledDeliveryOrigin';
```

Comment:
This constant already exist: https://github.com/oat-sa/extension-tao-delivery-rdf/blob/develop/model/DeliveryAssemblyService.php#L40